book/website/community-handbook/governance/gov-maintainance.md

```diff
@@ -0,0 +1,65 @@
+(ch-gov-maintenance)=
+# Maintenance level decision making
+
+Decisions taken at the maintenance level require extended engagement with the project and often require specialised interests or expertise.
+
+Decisions at this level are made by one of the approved Working Groups or Delivery Groups.
+
+All decisions must align to the 
+
+## Difference between a Working Group and a Delivery Group
+
+Both Working Groups and Delivery Groups are integral to sustaining *The Turing Way* as an open source project aligned with governance priciples of empowerment, transparency and inclusion.
+
+A Working Group is not accountable to any organisation outside of *The Turing Way* project itself.
+It is often - but not always - volunteer-led and seeks to have clear and achiveable pathways for individual contributors to become members if membership aligns with their interests and goals.
+
+A Delivery Group has accountability outside of *The Turing Way* project.
+There is often - but not always - funding associated with the activites.
+The accountability therefore corresponds to pre-existing funding agreements or employee objectives.
+Alternatively a Delivery Group may have accountability to another community or organisation.
+Delivery Groups are not as open as Working Groups but commit to being as open as possible and as closed as necessary.
```

Comment:
I like this explanation. Maybe it is worth adding a note that the purpose of Delivery Groups is so that we can support work which contributes to The Turing Way but which also has external stakeholders?